File: PlayerKitDemo/Classes/Commen/TableViewCell.m
//
// Created by majiancheng on 2018/1/3.
// Copyright (c) 2018 majiancheng. All rights reserved.
//

#import "TableViewCell.h"
#import "Dto.h"


@implementation TableViewCell {

}

- (void)loadData:(Dto *)dto {

}

+ (NSString *)identifier {
    return NSStringFromClass(self.class);
}

+ (CGFloat)height {
    return 44.0f;
}

@end

File: PlayerKitDemo/Classes/Commen/TableViewCell.swift
//
// Created by majiancheng on 2018/5/28.
// Copyright (c) 2018 majiancheng. All rights reserved.
//

import UIKit

class TableViewcell: UITableViewCell {
    func loadData(data: Dto) {

    }

    class func identifier() -> String {
        return String(describing: self)
    }

    class func height() -> CGFloat {
        return 44
    }

}